.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: ci
+on:
+  push:
+    branches: [main, release]
+  pull_request:
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    outputs:
+      web: ${{ steps.web.outputs.any_changed }}
+      server: ${{ steps.server.outputs.any_changed }}
+      worker: ${{ steps.worker.outputs.any_changed }}
```

Comment:
```suggestion
      web: ${{ steps.web.outputs.any_modified }}
      server: ${{ steps.server.outputs.any_modified }}
      worker: ${{ steps.worker.outputs.any_modified }}
```

`any_changed` will not detect deletions, which we should.

Ref:  [tj-actions/changed-files](https://github.com/tj-actions/changed-files)

<img width="1017" alt="image" src="https://github.com/user-attachments/assets/fe84b478-d4a5-4b20-bf85-856ab6570e3b">